InformationScripting/src/parsing/QueryBuilder.cpp

```diff
@@ -81,12 +81,13 @@ QPair<QStringList, QList<QChar>> QueryBuilder::split(const QString& text, const
 	QPair<QStringList, QList<QChar>> result;
 	QString currentString;
 	QVector<int> openScopes(3, 0);
-	for (int i = 1; i < text.size() - 1; ++i)
+	for (int i = 2; i < text.size() - 2; ++i)
```

Comment:
At the very least, define a static global constant at the beginning of this file = 2 and use that throughout.